server/datastore/mysql/migrations/tables/20220914154915_CreateHostDisksTable.go

```diff
@@ -0,0 +1,28 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220914154915, Down_20220914154915)
+}
+
+func Up_20220914154915(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+    CREATE TABLE host_disks (
+        host_id                      INT(10) UNSIGNED NOT NULL,
+        gigs_disk_space_available    FLOAT NOT NULL DEFAULT 0,
```

Comment:
Consider using a DECIMAL type here and for `percent_disk_space_available` as well - otherwise if we ever need to do some aggregations on any of these two columns we might run into rounding issues.